src/components/Login/Login.js

```diff
@@ -32,7 +32,7 @@ class Login extends Component {
     AsyncStorage.getItem('token', (err, result) => {
       if (err) {
         this.setState({ loaded: true });
-      } else if (err) {
+      } else if (result) {
```

Comment:
Travor found same thing. sorry my mistake.
Thanks to you guys, our project are getting better!
Thanks again!